### DISCUSSION TITLE
I want to connect another device to the server

### DISCUSSION THREAD

**Discussion by Legenfde** [stashapp/stash#3673] [Q&A]
I know there is a Wiki FAQ Page for this Topic but i dont get it to work it says that i need to find my IPv4 address which i have and then

i should go on this link on another device: http://server.ip.address.here:9999/

and i know i should replace the address.here and the server.ip but i try like every single combo i can imagine and it didnt work can someone pls help me ? or show me how to do it.

(Maybe i have to setup something, i dont really know how to but i would love to try)

**Comment by MrX292** [comment]
what os are you running on and is your firewall maybe blocking stash

**Reply by Legenfde** [reply]
Windows 11 and my Firewall should not Block something at least i didnt found something

**Reply by MrX292** [reply]
try adding an exception for stash

**Reply by Legenfde** [reply]
Its still doesnt work

**Reply by MrX292** [reply]
are both devices on the same network?

**Reply by Legenfde** [reply]
I was testing some Stuff the last two days and i couldnt send Messages because i had no time but i got it to work

**Reply by DogmaDragon** [reply]
Can you post what was the issue so I can mark the question as answered?

**Comment by feederbox826** [comment]
Resolved with no conclusion posted, Occam's razor says firewall
